.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Compile the protocol buffers, launch the backend with docker-compose
+    before: cd app
+    init: |
+      docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+      docker-compose -f docker-compose.yml -f docker-compose.gitpod.yml up --build
+
+  - name: Install and launch the frontend with yarn
+    before: cd app/frontend
+    init: |
+      yarn install
+      gp await-port 8888
+    command: REACT_APP_API_BASE_URL=$(gp url 8888) yarn start
+
+  - name: Instructions
+    before: cd app/
+    init: until (curl --head -s localhost:8888 localhost:8888 > /dev/null); do sleep 1; clear; echo "Waiting for backend & frontend to come up. This might take a few minutes!"; done
+    command: echo "Success! You can now launch the frontend by going to http://localhost:3000/"
```

Comment:
Also; I think this should be `gp url 3000`, not localhost?

Reply:
Ahh yes, I forgot to change it back when I was testing with the companion app, oops